src/analyses/base.ml

```diff
@@ -2013,7 +2013,7 @@ struct
         | [exp] ->
           begin match ThreadId.get_current ctx.ask with
             | `Lifted tid ->
-              let rv = eval_rv ctx.ask ctx.global ctx.local exp in
+              let rv = eval_rv ctx.ask ctx.global ctx.local exp |> VD.cast ~torg:(typeOf exp) Cil.voidPtrType in
```

Comment:
I'm a bit confused now, because here you added the cast also to `pthread_exit`. Was this an accidental change?

Reply:
>Was this an accidental change?

Yes, thanks for noting! I will remove the cast on master.